jobs/build-arch.Jenkinsfile

```diff
@@ -46,7 +46,7 @@ properties([
                    description: 'Force AWS AMI replication for non-production'),
       string(name: 'COREOS_ASSEMBLER_IMAGE',
              description: 'Override coreos-assembler image to use',
-             defaultValue: "coreos-assembler:main",
+             defaultValue: "",
```

Comment:
One weird thing I noticed while testing in stg is that for some reason changing the default value back to the empty string didn't take hold. Even after running the job multiple times, it would still be `coreos-assembler:main`. I ended up having to rerun the `seed` job with `FORCE=1`.